### DISCUSSION TITLE
RAG API with gptPlugins

### DISCUSSION THREAD

**Discussion by zakariaabboud** [danny-avila/LibreChat#3467] [Q&A]
Hello, I'am using gptPlugins endpoint and I can't use the rag api or the image vision with it. When using the azureOpenAI endpoint it's working fine. Is it just a parametre that I should ajust or plugins don't support rag api and image vision

**Comment by danny-avila** [comment]
The Plugins endpoint will be deprecated very soon in favor of a better system, which will have access RAG/vision/etc.

**Reply by zakariaabboud** [reply]
Ok, thanks for your response.

**Reply by zakariaabboud** [reply]
In this new version of LibreChat, the Plugins will be replaced with Assistants ? and if Yes, do the azure assistants in Librechat will support rag and vision in this version ?
